tests/test_translate.py

```diff
@@ -0,0 +1,18 @@
+from m4st.translate.model import T5TranslateModel
+
+eng_text = "Hi, my name is Bob."
+
+
+def test_t5():
+    t5 = T5TranslateModel("eng", "fra")
+    translated_text_t5 = t5(eng_text)
+    print(translated_text_t5)
+
+
+# NOTE: Ollama server must be running for this test to work, skipping it for now.
+# def test_llama32():
+#    llama32translate = OllamaTranslateModel()
+#    translated_text_llama32 = llama32translate(
+#        eng_text, source_lang_iso="eng", target_lang_iso="fra"
+#    )
+#    print(translated_text_llama32)
```

Comment:
would be neater to mark this for skipping with pytest rather than commenting it out.

Also neither of these tests have an assert, e.g. the first one could have an `assert translated_text_t5 is not None` or similar at least

Reply:
Yes, good point, I'll return to this after addressing the other comments.